src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -222,7 +229,7 @@ public static String getIndividualFullName(Entity entity) {
         return entity.getName().getFullNameNoTitle();
     }
 
-    public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions, OverrideFields overrideFields, ReferenceDataServiceHolder referenceData) throws InvalidMappingException {
+    public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions, OverrideFields overrideFields, ReferenceDataServiceHolder referenceData, String adjournLanguage) throws InvalidMappingException {
```

Comment:
Good so far but you need to set the adjournLangauge at this point, I would do it after the `if (nonNull(overrideFields)` and before the `if (isNull(hearingOptions)` block, returning adjournLanguage
```
if(nonNull(adjournLanguage)) {
    return adjournLanguage;
}
```